index.html

```diff
@@ -806,7 +806,7 @@
             </p>
             <div class="note">
               To avoid leaking information that could fingerprint the user, the
-              user agent SHOULD NOT assign a <var>callbackId</var> that uses any
+              user agent should not assign a <var>callbackId</var> that uses any
```

Comment:
The HTML spec as well as some other specs use the "is expected to" language to avoid ambiguity between capitalized and lowercase RFC term. In this case:

"the user agent is expected to not assign"